src/seedu/addressbook/AddressBook.java

```diff
@@ -1180,7 +1180,7 @@ private static String removePrefixSign(String s, String sign) {
      * @return split by whitespace
      */
     private static ArrayList<String> splitByWhitespace(String toSplit) {
-        return new ArrayList(Arrays.asList(toSplit.trim().split("\\s+")));
+        return new ArrayList(Arrays.asList(toSplit.toLowerCase().trim().split("\\s+"))); //1kLoc change toSplit, toLowerCase
```

Comment:
i guess this would mess up the way data is stored/displayed as discussed in the session.